<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="fixed_base" default="true"/>
    <node pkg="mppi_manipulation" type="raisim_rendering_node" name="raisim_rendering_node" output="screen">
      <param name="fixed_base" value="$(arg fixed_base)"/>
    </node>
</launch>
